moVirt/src/main/java/org/ovirt/mobile/movirt/ui/vms/VmDetailActivity.java

```diff
@@ -226,23 +226,20 @@ void start() {
     }
 
     @OptionsItem(R.id.action_stop)
-    @UiThread
     void stop() {
         DialogFragment confirmDialog = ConfirmDialogFragment
                 .newInstance(ACTION_STOP_VM, getString(R.string.dialog_action_stop_vm));
         confirmDialog.show(getFragmentManager(), "confirmStopVM");
```

Comment:
I couldn't find documentation for this (same as the 2nd comment), but i found many examples on the Internet which use @OptionsItem (ie onOptionsItemSelected ) as UiThread (We also do). I also debugged it and concluded it runs on the main thread and we do not need @UiThread annotation here.